conans/client/conf/__init__.py

```diff
@@ -47,6 +47,7 @@
     Arduino:
         board: ANY
     Emscripten:
+    QNX:
```

Comment:
This is ok. But is that enough for managing the QNX operating system? Take a look at the AIX pull request changes. https://github.com/conan-io/conan/pull/4546
I'm totally ok adding QNX to the yml if you tell me no further changes are really required. but please take a look at it.

Reply:
Hi,
at the moment I am using a customized ```settings.yml``` with:
```yaml
os:
    QNX:
        version: ["6.4", "6.5", "7.0"]
```

At the moment it is not forseeable if this is neccessary. But QNX is a real specialized system that one can configure down to the supported API etc. and I cannot safely say that without version differentiation everything will do fine in the future.

But as I and every QNX user have a customized ```settings.yml``` for this anyway, I would go with minimal additions to default Conan settings for the moment. We can add settings in the future.